src/lib/github/addIssuesToRepo.ts

```diff
@@ -11,21 +12,33 @@ export default async function addIssuesToRepo (context: Context, newIssues: Issu
   const octokit = context.github;
   const { owner, repo } = getBasicRepoProps (context);
   const ghIssues = await getAllRepoIssues(context);
+
   newIssues.forEach(async ({title, body, authors}) => {
     const assignees = uniq(authors);
+
     const currentGHIssue = find(ghIssue => ghIssue.title === title, ghIssues);
+    const newGHIssueComments = currentGHIssue.comments.includes("- [x]") || !(currentGHIssue.comments === body)
+     ? newIssueBody(currentGHIssue.comments, body)
+     : currentGHIssue.comments;
+
     const fields = {
       owner,
       repo,
-      body,
+      newGHIssueComments,
       title,
       labels: ["GH-TODO-BOT"],
       assignees,
       ...(currentGHIssue ? { number: currentGHIssue.number } : {})
     };
+
     return currentGHIssue
       ? octokit.issues.createComment(fields)
       : octokit.issues.create(fields);
   });
 
 }
+
+function newIssueBody(currentIssueBody: string[], newIssueBody: any): string[] {
```

Comment:
This function seems to imply `newIssueBody` is of type `string[]` which is not the case.
`body` is a string which is created by joining an array of issue comments. i.e its something like.
`- [ ] some comment text \n  -[ ] some other comment text`

Reply:
@epicallan , i get your point, but could changing the `type` of body as mentioned earlier [here](https://github.com/freelyformd/github-todos/pull/70#discussion_r232781169) cause more issues?

Reply:
Yes, it should remain as `string`.